dvclive/lgbm.py

```diff
@@ -0,0 +1,17 @@
+import dvclive
+
+
+class DvcLiveCallback:
+    def __init__(self, model_file=None):
+        super().__init__()
+        self.model_file = model_file
+
+    def __call__(self, env):
+        for eval_result in env.evaluation_result_list:
+            metric = eval_result[1]
+            value = eval_result[2]
+            dvclive.log(metric, value)
+        dvclive.next_step()
+
+        if self.model_file and env.iteration == env.end_iteration - 1:
+            env.model.save_model(self.model_file)
```

Comment:
Would this only sve the model in the last iteration?? Would it make sense to save it at the end of each iteration, like in xgb:

https://github.com/iterative/dvclive/blob/master/dvclive/xgb.py#L12

Regardless, [dvclive.next_step](https://dvc.org/doc/dvclive/api-reference/next_step) should be called after the model saving in order to work as intended when dvclive is used with DVC checkpoints.

```suggestion
        if self.model_file:
            env.model.save_model(self.model_file)
        dvclive.next_step()
```